### PR TITLE
Fix binomial primary decomposition

### PR DESCRIPTION
**Pull request by CarloSircana**

### CODE REVIEW

**Comment by thofma** [comment]
Could you add a test?

**Comment by CarloSircana** [comment]
This code is not tested at the moment, as it is also calling an external program (markov4ti2)...

File: examples/TranslateBinomial2.jl [hunk 1]
```diff
@@ -542,6 +560,9 @@ function cellular_associated_primes(I::MPolyIdeal)
 
   #construct the ideal (x_i \mid i \in \Delta^c)
   gi = elem_type(R)[Variables[i] for i = 1:nvars(R) if !(i in cell[2])]
+  if isempty(gi)
```
Comment (thofma) [inline examples/TranslateBinomial2.jl:563]: Out of curiosity. Do ideals with zero generators behave badly?

Comment (CarloSircana) [reply examples/TranslateBinomial2.jl:563]: The constructor is missing.

**Comment by thofma** [comment]
Since there are no tests, I guess this is fine.

**Pull request merged**